src/components/PostEditor/PostEditor.connector.js

```diff
@@ -62,7 +61,7 @@ export function mapStateToProps (state, props) {
   const networkSlug = getRouteParam('networkSlug', null, props)
   const topic = getTopicForCurrentRoute(state, props)
   const topicName = get('name', topic)
-  const postTypeContext = getPostTypeContext(null, props) || getQuerystringParam('t', null, props)
```

Comment:
hmm what was the t query param used for..??

Reply:
@tibetsprague I'm not sure but it didn't seem to break things when I took it out